src/Card/CardHeader.js

```diff
@@ -0,0 +1,69 @@
+import CardCheckBox from './CardCheckBox';
+import './Card.css';
```

Comment:
Would advice you to play with styling so icons and fields and not flipping in different Card modes.